### PR TITLE
Carbon Interpreter

### PR DESCRIPTION
**Pull request by blankse**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets |

Date Fields needs Carbon as value

### CODE REVIEW

File: src/ImportDefinitionsBundle/Interpreter/CarbonInterpreter.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+{
+    use DataSetAwareTrait;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function interpret(Concrete $object, $value, Mapping $map, $data, DefinitionInterface $definition, $params, $configuration)
+    {
+        if ($value) {
+            return new \Carbon\Carbon($value);
```
Comment (aarongerig) [inline src/ImportDefinitionsBundle/Interpreter/CarbonInterpreter.php]: Qualifier is unnecessary and can be replaced with an import.

```php
use Carbon\Carbon;
...
return new Carbon($value);
```

**Comment by aarongerig** [comment]
Thanks for the PR! I just added a comment.

File: src/ImportDefinitionsBundle/Interpreter/CarbonInterpreter.php (hunk 1 above)
Comment (blankse) [reply src/ImportDefinitionsBundle/Interpreter/CarbonInterpreter.php]: I changed it. I have oriented to QuantityValueInterpreter :)

**Comment by dpfaffenbauer** [comment]
thanks

**Pull request merged**